### PR TITLE
FastpackTranspiler: Export default class. Fixes #65

### LINKED ISSUE #65: Transpiling `export default class` fails when class has static properties or method decorators
Examples (produce different output, but both are incorrect):
``` javascript
export default class X {
  static prop = 1;
}
```

``` javascript
export default class {
  static prop = 1;
}
```

I have not tested this one, but very likely to fail as well:
``` javascript
export class X {
  static prop = 1;
}
```

### PR DESCRIPTION
**Pull request by zindel**

### CODE REVIEW

File: FastpackTranspiler/Class.ml [hunk 1]
```diff
@@ -244,20 +246,75 @@ let transpile {Context. require_runtime; _ } program =
...
-        | cls, Some (_, name), statics, classDecorators, decorators ->
+          [
+            loc,
+            S.ExportDefaultDeclaration {
+              export with
+              declaration = S.ExportDefaultDeclaration.Declaration (
+                  Loc.none,
+                  S.ClassDeclaration cls
+                )
+            }
```
Comment (TrySound) [inline FastpackTranspiler/Class.ml]: why curly brackets are aligned, but parentheses shifted?

Comment (zindel) [reply FastpackTranspiler/Class.ml]: In fact, I used `ocp-indent` this time around. I this we should convert all the code to use it at a certain point

Comment (TrySound) [reply FastpackTranspiler/Class.ml]: Just a bit weird and inconsistent from my point of view.

Comment (zindel) [reply FastpackTranspiler/Class.ml]: Well, I tend to agree with you, and in fact, have been ignoring the `ocp-indent` so far. But, to be honest, I think we should be adopting some automated code-styling. At the moment, I see the `ocp-indent` as the less intrusive option comparing to the `ocamlformat`

Comment (TrySound) [reply FastpackTranspiler/Class.ml]: Is there an option for this in ocp-indent?

**Pull request merged**